packages/@dcl/inspector/src/setupTests.ts

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom'
```

Comment:
can you ditch CRA in benefit of `estrella`? It will remove hundreds of MBs in deps and will make build times 100x faster

Reply:
https://github.com/decentraland/holodeck/blob/main/build.js

Reply:
the thing with `estrella` is that I think that it's best suited for `.ts` projects (thinking about CLI 🤔 ) since otherwise we will have to mantain the build against all the file types we will have on the project (svg, png, css, html, etc). CRA does all that stuff for us + it's today's (unfortunately) standard way to create react apps.
If we don't care about that, I will make the changes 👍

Reply:
estrella is only for the react project, it is the best package we've found to make the SDK work on react applications. the webviews from the editor, kernel and holodeck use it

Reply:
Hey, sorry I just saw this. I would not recommend using `estrella` to make a production build of any webapp, CRA already handles (and optimizes a lot) this for us. We use estrella to bundle tyspecript files and treeshake them, it does that amazingly fast and easily, but outputting an optimized build for a webapp (css, media files, etc), that's a whole different thing. 

We don't care about size of dependencies here (they are not included in the artifact) as much as size/efficiency of the production build, since we are not building a library but a webapp.